packages/reqresp/test/utils/response.ts

```diff
@@ -0,0 +1,33 @@
+import {pipe} from "it-pipe";
+import {allForks} from "@lodestar/types";
+import {responseEncodeError, responseEncodeSuccess} from "../../src/encoders/responseEncode.js";
+import {RespStatus} from "../../src/interface.js";
+import {EncodedPayload, EncodedPayloadType, ProtocolDefinition} from "../../src/types.js";
+import {ResponseChunk} from "../fixtures/encoders.js";
+import {beaconConfig} from "../fixtures/messages.js";
+import {blocksToReqRespBlockResponses} from "../utils/block.js";
+import {arrToSource} from "../utils/index.js";
+
+export async function* responseEncode(
```

Comment:
What is the purpose of this function?

Reply:
@dapplion  To be used in the tests for the `test/unit/encoders/responseEncode.test.ts`. Particularly used to same assertions data as it exists earlier.

Reply:
Simplified the `responseEncode` method. Its a utility function for tests, so definitely would not used in production.